src/enums/summaryActions.ts

```diff
@@ -1,5 +1,6 @@
 export enum SummaryActions {
-  SAVE_RESUME_LATER,
-  DELETE_ALL,
-  CONFIRM
+  BACK,
+  CANCEL,
+  CONFIRM,
+  SAVE_RESUME_LATER
 }
```

Comment:
I suggest this be renamed to FeeSummaryActions, so it's not confused with our Actions component.

Reply:
K
Updated